openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -267,12 +267,22 @@ def write_project_to_op(project: dict, dbcon: AvalonMongoDB) -> UpdateOne:
         # Update Zou
         gazu.project.update_project(project)
 
+    all_kitsu_projects_names = [project['name'] for project in gazu.project.all_projects()]
+    all_kitsu_open_projects_names = [project['name'] for project in gazu.project.all_open_projects()]
+    all_kitsu_closed_projects_names = [project for project in all_kitsu_projects_names if project not in all_kitsu_open_projects_names]
+
+    if project_name in all_kitsu_closed_projects_names:
+        project_status = False
+    else:
+        project_status = True
```

Comment:
It seems that the closed projects on Kitsu are not synchronized, so we have to make a difference between all the projects and the open projects to see whether a project is open or not

Reply:
[Looking at the logic of the sync all projects it wouldn't ever mark a project inactive](https://github.com/ynput/OpenPype/blob/eea058be0d984eb657d8c3c9ee40b958a9debc83/openpype/modules/kitsu/utils/update_op_with_zou.py#L341) because it'd never start and try to sync the project to begin with. As such, I think the way you're trying to work around it isn't the way to go. Right?

Maybe the way is that syncing all projects should iterate ALL projects including inactive ones, but for the inactive ones only set the status if the project already exists in OpenPype - if not, then ignore it?

Reply:
So you think instead of modifying the write_project_to_op function, we should modify sync_all_projects to include all projects and not just the open ones ?